index.bs

```diff
@@ -390,8 +390,14 @@ opaque sequences of 16-bit code units.
   can store.
 </aside>
 
-A <dfn>sorted name list</dfn> is a list containing [=/names=]
-sorted in ascending order by 16-bit code unit.
+<div algorithm>
+
+To <dfn>create a sorted name list</dfn> from a [=/list=] |names|, run these steps:
```

Comment:
Up to you, but I'd make the `lt="creating a sorted name list"`, so that references can be "Return the result of creating a sorted name list…" rather than "Return the result of running create a sorted name list…", but maybe the latter is more consistent with this spec.

Reply:
Good idea. Turns out Bikeshed can infer "creating" from "create", no need to even change the algorithm definition.